hack/install-config.yaml.template

```diff
@@ -6,13 +6,15 @@ compute:
   name: worker
   platform:
     aws:
-      type: m5.2xlarge
+      type: m6i.xlarge
   replicas: 3
 controlPlane:
   architecture: amd64
   hyperthreading: Enabled
   name: master
-  platform: {}
+  platform:
+    aws:
+      type: m6i.2xlarge
```

Comment:
m5.2xlarge 	8cpu 	32Gb
m6i.xlarge 	4cpu 	16Gb

This is actually smaller than before. Also, why we're changing the machine size in this ticket?

Reply:
I've removed this, focusing the patch on the log configuration/